src/aznfswatchdog

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash 
+
+#
+# How often does the watchdog look for unmounts and/or IP address changes for 
+# Blob endpoints. 
+#
+MONITOR_INTERVAL_SECS=5
+
+# Load common aznfs helpers.
+. /opt/microsoft/aznfs/common.sh
+
+# Create the MOUNTMAP if it does not exist.
+if [ ! -f $MOUNTMAP ]; then
+    touch $MOUNTMAP
+fi
+
+# Create the LOGFILE if it does not exist.
+if [ ! -f $LOGFILE ]; then
+    touch $LOGFILE
+fi
+
+# 
+# Watchdog for monitoring unmounts and more importantly change in blob endpoint
+# addresses possibly as a result of migration. 
+#
+while :; do
+    sleep $MONITOR_INTERVAL_SECS
+
```

Comment:
here we should have a call to
reconcile_mountmap

this function should create a mountmap file from findmnt and iptables o/p..
you can leave that function empty for now and later we will fill it in subsequent revisions

Reply:
For now, I have added a TODO.